public/app/features/scopes/dashboards/types.ts

```diff
@@ -6,14 +6,20 @@ export interface SuggestedDashboard {
   items: ScopeDashboardBinding[];
 }
 
-export interface SuggestedDashboardsFolder {
+export interface SuggestedNavigation {
   title: string;
-  expanded: boolean;
-  folders: SuggestedDashboardsFoldersMap;
-  dashboards: SuggestedDashboardsMap;
+  url: string;
+  // Used for testid and keys
+  name: string;
```

Comment:
```suggestion
  // Used for testid and keys
  id: string;
```
🤔 this is the dashboard name or the ScopeNavigation name

Reply:
Yeah that probably makes sense, the `name` is confusing here.